test/onnx_import.in.cpp

```diff
@@ -2375,3 +2375,147 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, model_quant_conv_linear)
         execute<std::uint8_t, std::int8_t>(function, inputs, "${BACKEND_NAME}")};
     EXPECT_TRUE(test::all_close(expected_output.front(), outputs.front()));
 }
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_quant_conv_linear_2d)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/qlinear_conv_2d.prototxt"));
+
+    auto x =
+        read_binary_file<uint8_t>(file_util::path_join(SERIALIZED_ZOO, "onnx/qlinearconv2d/x.bin"));
```

Comment:
Would be nice if you could move all these binary files to `test/files/onnx/` directory. Then just use `TEST_FILES` symbol instread of `SERIALIZED_ZOO`:
```suggestion
        read_binary_file<uint8_t>(file_util::path_join(TEST_FILES, "onnx/qlinearconv2d/x.bin"));
```